README.md

```diff
@@ -1176,6 +1176,35 @@ class TestWidget extends WidgetBase<WidgetProperties> {
 }
 ```
 
+#### Resize
+
+The resize observer meta uses the latest [`ResizeObserver`](https://wicg.github.io/ResizeObserver/) within your widgets. This is currently supported within `Chrome 64+` and supported via [polyfill](https://github.com/WICG/ResizeObserver/issues/3).
```

Comment:
within your widgets -> within Dojo-based widgets

Reply:
This is currently supported within `Chrome 64+` and supported via -> Native browser support is currently provided by `Chrome 64+` and other Dojo supported browsers work via